### PR TITLE
Added solid and stroked rounded rect, based of 0.8.6 implementation

### PR DESCRIPTION
**Pull request by ryanbartley**

### CODE REVIEW

**Comment by ryanbartley** [comment]
Hold on this. I'm updating the implementation of Solid to allow for texCoords and Normals.

File: src/cinder/gl/draw.cpp [hunk 1]
```diff
@@ -880,6 +1052,7 @@ void drawSolidCircle( const vec2 &center, float radius, int numSegments )
 		if( normals )
 			normals[s+1] = vec3( 0, 0, 1 );
 		t += tDelta;
+		cout << texCoords[s+1] << endl;
```
Comment (andrewfb) [inline src/cinder/gl/draw.cpp]: Looks like a stray cout snuck in here...

Comment (ryanbartley) [inline src/cinder/gl/draw.cpp]: whoops! I'll take care of that. Also, I'll update the Stroked Rounded Rect as well.

File: src/cinder/gl/draw.cpp [hunk 2]
```diff
@@ -718,6 +833,63 @@ void drawStrokedRect( const Rectf &rect, float lineWidth )
...
+	int posLoc = curGlslProg->getAttribSemanticLocation( geom::Attrib::POSITION );
+	if( posLoc >= 0 ) {
+		enableVertexAttribArray( posLoc );
+		vertexAttribPointer( posLoc, 2, GL_FLOAT, GL_FALSE, 0, (void*)0 );
+	}
+	ctx->getDefaultVao()->replacementBindEnd();
+	ctx->setDefaultShaderVars();
+	ctx->drawArrays( GL_LINE_LOOP, 0, tri );
+	ctx->popVao();
+	delete [] verts;
```
Comment (richardeakin) [inline src/cinder/gl/draw.cpp]: i know this is the from the version in 0.8.6, but could you use unique_ptr, or better yet std::vector, to manage the memory without an explicit delete? You can [reserve()](http://en.cppreference.com/w/cpp/container/vector/reserve) the memory it needs up front to make it the same in efficiency.

Thanks for adding these though, they're one of the last missing ones remaining.

Comment (ryanbartley) [inline src/cinder/gl/draw.cpp]: Yeah. That's what I meant about updating this method like the drawsolid version.

**Pull request merged**